cli/cli-client/src/main/java/org/infinispan/cli/shell/Completer.java

```diff
@@ -51,12 +50,13 @@ public void complete(CompleteOperation op) {
                      break;
                   }
                }
+               List<String> candidates = new ArrayList<>();
                addPrefixMatches(procCmd.getCurrentArgument(), command.getOptions(), candidates);
                command.complete(context, procCmd, candidates);
```

Comment:
I wanted to abstract this: otherwise I need to add an explict dep to AEsh apis in the commands